src/AWS.Distro.OpenTelemetry.AutoInstrumentation/Plugin.cs

```diff
@@ -81,48 +96,60 @@ public void TracerProviderInitialized(TracerProvider tracerProvider)
 
             tracerProvider.AddProcessor(AttributePropagatingSpanProcessorBuilder.Create().Build());
 
-            string? intervalConfigString = System.Environment.GetEnvironmentVariable(MetricExportIntervalConfig);
-            int exportInterval = DefaultMetricExportInterval;
-            try
-            {
-                int parsedExportInterval = Convert.ToInt32(intervalConfigString);
-                exportInterval = parsedExportInterval != 0 ? parsedExportInterval : DefaultMetricExportInterval;
-            }
-            catch (Exception)
+            // We want to be adding the exporter as the last processor in the traceProvider since processors
+            // are executed in the order they were added to the provider.
+            if (this.IsLambdaEnvironment() && !this.HasCustomTracesEndpoint())
```

Comment:
What if `this.IsLambdaEnvironment()` = true but `!this.HasCustomTracesEndpoint()` = false? Is it a valid case?

Reply:
This means we are in a lambda environment but the customer explicitly set the custom traces endpoint which means that they want to use their own customer collector and this case, we don't enable the udp exporter since it's only used for the xray daemon.